app/uk/gov/hmrc/apidocumentation/views/PrivacyView.scala.html

```diff
@@ -72,13 +73,13 @@ <h2 class="heading-large">What personal information do we hold about you?</h2>
    </section>
    <section>
        <h2 class="heading-large">How we use and share your data</h2>
-       <p>We use this information to create and administer user accounts on the Developer Hub. 
-           The information you give us will be used in a customer relationship management tool 
-           to help us manage our customer service obligations. We don’t share this information 
+       <p>We use this information to create and administer user accounts on the Developer Hub.
```

Comment:
Has this removed spaces from the output text?

Reply:
I think it should be fine as the next text is in a different line, so it will probably be rendered with a space as before. But we should definitely have a look at this view as part of QA and make sure it renders the way we wanted.

Reply:
+1

Reply:
We've checked and it's fine when it's rendered.